### PR TITLE
22751: add the `regions` definition in the info/*.json endpoints

### PR DESCRIPTION
**Pull request by fersagui**
Redmine: [22751: Adjust info/*.json endpoints to a list of regions](https://redmine.buzzhives.com/issues/22751)
Satapp PR: https://github.com/skedgo/skedgo-java/pull/7211

### CODE REVIEW

File: docs/specs/tripgo.openapi.yaml [hunk 1]
```diff
@@ -135,20 +135,26 @@ paths:
...
       requestBody:
         content:
           application/json:
             schema:
               required:
-              - region
+              - regions
               type: object
               properties:
                 region:
```
Comment (fersagui) [inline docs/specs/tripgo.openapi.yaml:149]: Should we mark this as deprecated ?

Comment (nighthawk) [reply docs/specs/tripgo.openapi.yaml:149]: Yes, please deprecate the old fields

**Comment by nighthawk** [comment]
@fersagui Didn't we say that the response also changes to indicate which region(s) an operator is in?

File: docs/specs/tripgo.openapi.yaml (hunk 1 above)
Comment (fersagui) [reply docs/specs/tripgo.openapi.yaml:149]: They are all deprecated right now

**Comment by fersagui** [comment]
> @fersagui Didn't we say that the response also changes to indicate which region(s) an operator is in?

Yes, I have added this in https://github.com/skedgo/tripgo-api/pull/254/commits/5b008cfa000c605822f865c6b7a3857ccf00e22b

**Review by nighthawk — APPROVED** [review]

**Pull request merged**